tests/autofix_files/async910.py.diff

```diff
@@ -205,8 +205,16 @@
 
 
  async def foo_comprehension_3():
-@@ x,3 x,4 @@
+@@ x,6 x,7 @@
```

Comment:
Ooooh after some confusion I think I figured out why this line in the diff output got changed. It's because there's now context after the changed line in `await_everywhere_except_gen_target` (i.e. two empty lines and `# Issue #226`), so now there's 3 more lines in the diff output.